core/extension/compat_hashes_81386.h

```diff
@@ -0,0 +1,57 @@
+/**************************************************************************/
+/*  compat_hashes_81386.h                                                 */
```

Comment:
I would remove the PR number in the file/class name and put it into the name of the initialize method instead, as that is the only part that should need to be duplicated if this ever becomes relevant again.

(Assuming that there will be more cases where the hash has to change for some reason)

Reply:
I was imagining that we'd make a new class if we had to do something like this again. But you're suggesting we could just re-use this same class but have individual initialize methods for each issue where we broke the hashing algorithm?

I'm a little reluctant to do this, given that we don't know what another hash breaking issue would look like. If there was another hash break, and it meant all or some the `current_hash`es listed here were now wrong, we'd have have to either (1) update all or some of the `current_hash`es for the new values (which would be a pain without scripting this process again) or (2) have the mapping for the new hash breakage run _after_ the mapping here (so, we could leave all the code added here exactly the same, but then do another mapping on the mapped value). Depending on how many hashes we're talking about, option nr 1 or 2 may appear more attractive at the time.

So, I'd prefer to leave this as it is for now, and consider changing it if/when we have to break the hashes again. But if everyone feels strongly, I'd be fine switching this around (since we could equally change it back to the way it is now if it made sense when the next issue came up :-)).

Reply:
Alright, I'll rename it in my next push :-)

Reply:
Renamed!